README.md

```diff
@@ -395,7 +422,35 @@ fn search(path: &Path, cost_so_far: usize, best_cost: &Arc<AtomicUsize>) {
 Now in this case, we really WANT to see results from other threads
 interjected into our execution!
 
-## License
+## Semver policy, the rayon-core crate, and unstable features
+
+Rayon follows semver versioning. However, we also have APIs that are
+still in the process of development and which may break from release
+to release -- those APIs are not subject to semver. To use them,
+you have to set the cfg flag `rayon_unstable`. The easiest way to do this
+is to use the `RUSTFLAGS` environment variable:
+
+```
+RUSTFLAGS='--cfg rayon_unstable' cargo build
+```
+
+Note that this must not only be done for your crate, but for any crate
+that depends on your crate. This infectious nature is intentional, as
+it serves as a reminder that you are outside of the normal semver
+guarantees. **If you see unstable APIs that you would like to use,
+please open an issue about stabilizing them!**
```

Comment:
It occurs to me that the infectious nature is actually a double-edged sword.  Suppose one of my dependencies is happily using stable rayon-0.8 behind the scenes, but then I want to enable unstable rayon-0.9 / rayon-core-1.2 for my own purposes.  Broadly enabling `--cfg rayon_unstable` will break that stable 0.8-using crate too.